tenant_schemas_celery/app.py

```diff
@@ -18,8 +18,9 @@ def get_schema_name_from_task(task, kwargs):
         return kwargs.pop("_schema_name", None)
 
     # In some cases (like Redis broker) headers are merged with `task.request`.
-    task_headers = task.request.headers or task.request
-    return task_headers.get("_schema_name")
+    if "_schema_name" in task.request.headers:
```

Comment:
It's good to check whether the `headers` attr is set. Previous expression handled such cases.

```suggestion
    if task.request.headers and "_schema_name" in task.request.headers:
```